src/main/java/com/tabnine/binary/requests/autocomplete/CompletionPostprocess.kt

```diff
@@ -0,0 +1,48 @@
+package com.tabnine.binary.requests.autocomplete
+
+fun postprocess(request: AutocompleteRequest, result: AutocompleteResponse, tabSize: Int) {
```

Comment:
I think `tabSize` is constant, and we should calculate the entire regex only once. As far as I know, compiling regexes could have a performance penalty.

Reply:
And it is best if we can calculate it on plugin start.

Reply:
no, depends on the project / file you're working on - can also be changed by the user..